packages/wallet-backend-update/src/rapyd/rapyd-client.ts

```diff
@@ -0,0 +1,248 @@
+import axios, { AxiosError, AxiosResponse } from 'axios'
+import crypto from 'crypto-js'
+import { Logger } from 'winston'
+import { Env } from '@/config/env'
+
+interface IRapydClient {
+  createWallet(
+    wallet: RapydWallet
+  ): Promise<AxiosResponse<RapydResponse<RapydWallet>>>
+  updateProfile(
+    profile: RapydProfile
+  ): Promise<AxiosResponse<RapydResponse<RapydWallet>>>
+  verifyIdentity(
+    req: RapydIdentityRequest
+  ): Promise<AxiosResponse<RapydResponse<RapydIdentityResponse>>>
+  depositLiquidity(
+    req: RapydDepositRequest
+  ): Promise<AxiosResponse<RapydResponse<RapydDepositResponse>>>
+  holdLiquidity(
+    req: RapydHoldRequest
+  ): Promise<AxiosResponse<RapydResponse<RapydHoldResponse>>>
+  releaseLiquidity(
+    req: RapydReleaseRequest
+  ): Promise<AxiosResponse<RapydResponse<RapydReleaseResponse>>>
+  transferLiquidity(
+    req: RapydTransferRequest
+  ): Promise<AxiosResponse<RapydResponse<RapydSetTransferResponse>>>
+  getAccountsBalance(
+    walletId: string
+  ): Promise<AxiosResponse<RapydResponse<RapydAccountBalance[]>>>
+  getDocumentTypes(
+    country: string
+  ): Promise<AxiosResponse<RapydResponse<RapydDocumentType[]>>>
+  getCountryNames(): Promise<AxiosResponse<RapydResponse<RapydCountry[]>>>
+}
+
+interface RapydClientDependencies {
+  logger: Logger
+  env: Env
+}
+
+export class RapydClient implements IRapydClient {
+  constructor(private deps: RapydClientDependencies) {}
+
+  public createWallet(
+    wallet: RapydWallet
+  ): Promise<AxiosResponse<RapydResponse<RapydWallet>>> {
+    return this.post('user', JSON.stringify(wallet))
```

Comment:
I think `post`, `put` should accept a generic. The return type is still any in this case.